android/TodokTodok/app/src/main/java/com/team/todoktodok/presentation/view/discussiondetail/vm/DiscussionDetailViewModel.kt

```
@@ -5,23 +5,25 @@ import androidx.lifecycle.MutableLiveData
 import com.team.todoktodok.presentation.core.event.SingleLiveData
 import com.team.todoktodok.presentation.view.discussiondetail.DiscussionDetailUiEvent
+import com.team.todoktodok.presentation.view.discussiondetail.model.DiscussionUiModel
 import kotlinx.coroutines.launch
 
 class DiscussionDetailViewModel(
     savedStateHandle: SavedStateHandle,
     private val discussionRepository: DiscussionRepository,
+    private val tokenRepository: TokenRepository,
 ) : ViewModel() {
     private val discussionId =
         savedStateHandle.get<Long>(KEY_DISCUSSION_ID) ?: throw IllegalStateException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)